src/libos/crates/async-rt/src/scheduler/entity.rs

```diff
@@ -171,4 +173,22 @@ impl SchedState {
     pub(crate) fn clear_enqueued(&self) {
         self.is_enqueued.store(false, Release);
     }
+
+    /// Set is_yielded to true, returning the old value.
+    ///
+    /// The is_yielded state helps the scheduler to
+    /// choose appropriate position of runqueue for task
+    pub(crate) fn set_yielded(&self) -> bool {
+        self.is_yielded.swap(true, Acquire)
+    }
+
+    /// Get is_yielded
+    pub(crate) fn is_yielded(&self) -> bool {
```

Comment:
`is_yielded` and `clear_yielded` are unnecessary

Reply:
Done.